.dependabot/config.yml

```diff
@@ -523,3 +523,14 @@ update_configs:
     - match:
         dependency_name: demisto/*
         update_type: all
+  - package_manager: python
```

Comment:
i dont think you need to add it manually
@glicht can you confirm ?

Reply:
It is ok. The create docker image script adds this. We need this for dependabot to detect updates.